test/nl/surf/eduhub/validator/service/jobs/client_test.clj

```diff
@@ -23,6 +24,7 @@
     (is (= http-status/ok status))
     body))
 
+;; TODO: Explain what this does and why
```

Comment:
This is a thread-safe mock implementation of a job queue. It will atomically return the top item on the queue and remove that item from the queue.